freeipa.doap.rdf

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en" xmlns="http://usefulinc.com/ns/doap#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:foaf="http://xmlns.com/foaf/0.1/">
+    <Project rdf:about="https://www.freeipa.org">
+        <created>2005-06-01</created>
+        <name>FreeIPA Project</name>
+        <shortname>FreeIPA</shortname>
+        <homepage rdf:resource="https://www.freeipa.org" />
+        <shortdesc>FreeIPA is the upstream open-source project for Red Hat Identity Manager</shortdesc>
+        <description>FreeIPA is an integrated security information management solution combining Linux (Fedora), 389 Directory Server, MIT Kerberos, Chronyd, DNS, Dogtag (Certificate System). It consists of a web interface and command-line administration tools.</description>
+        <bug-database rdf:resource="https://pagure.io/freeipa/" />
+        <mailing-list rdf:resource="https://www.freeipa.org/page/Contribute" />
+        <mailing-list>freeipa-interest@redhat.com</mailing-list>
+        <mailing-list>freeipa-devel@lists.fedorahosted.org</mailing-list>
+        <mailing-list>freeipa-users@lists.fedorahosted.org</mailing-list>
+        <download-page rdf:resource="https://www.freeipa.org/page/Downloads" />
+        <programming-language>C</programming-language>
+        <programming-language>Python</programming-language>
+        <release>
+            <Version>
+                <name>stable</name>
+                <branch>4.6.90.pre2</branch>
+                <created>2018-05-15</created>
```

Comment:
You should not hardcode the branch and created values here. Which means this file needs to be renamed freeipa.doap.rdf and dynamically built by the make system.

Reply:
I thought about dynamic creation, too. I assume the file is read from source repos, which means our git needs to contain a valid DOAP file with all information pre-populated (unverified assumption).

Reply:
branch could always just be master, ipa-4-6, whatever with the "created" being when the branch was done. I dunno about stable and master :-)

Reply:
@tiran @rcritten 
Agreed that we shouldn't have hardcorded values. So does this mean we should create the freeipa.doap.rdf from spec file rather.

Reply:
@menonsudhir Please research how tools are consuming the file and if the file format is still used. I don't want to waste your time on a meta data format, that might be outdated or obsolete.

Reply:
And I'm swinging back the other way on this. It may very well be ok to be static if the branch is simply master and doesn't reflect a version. The date would probably be the start date of the project since master started then. How useful this is is another matter.